src/member/member.controller.ts

```diff
@@ -10,6 +10,8 @@ import {
   UseGuards,
   Delete,
   Param,
+  HttpStatus,
+  HttpException,
```

Comment:
I think it will be fine to use APIException to throw error , this function can return 400 when api get error

Reply:
Ok, i have removed HttpStatus /  HttpException in the import and use APIException when an error occurs